oe_theme.theme

```diff
@@ -1522,3 +1523,64 @@ function oe_theme_preprocess_page(array &$variables): void {
     'id' => ['main-content'],
   ]);
 }
+
+/**
+ * Implements hook_preprocess().
+ */
+function oe_theme_preprocess_oe_organisation(&$variables) {
+  $cacheability = CacheableMetadata::createFromRenderArray($variables);
+
+  $entity = $variables['entity'];
+  // Prepare fields.
+  $fields_values = ['name', 'oe_acronym'];
+  foreach ($fields_values as $field) {
+    if (!$entity->get($field)->isEmpty()) {
```

Comment:
You can do $variables[$field] = ''; before the if instead of else to not add cyclomatic complexity